tools/handshake-runner/handshake-runner.cpp

```diff
@@ -367,8 +369,8 @@ bool executeStdOp(mlir::Operation &op, std::vector<Any> &inValues,
 }
 
 void executeFunction(mlir::FuncOp &toplevel,
-                     DenseMap<mlir::Value , Any> &valueMap,
-                     DenseMap<mlir::Value , double> &timeMap,
+                     llvm::DenseMap<mlir::Value , Any> &valueMap,
+                     llvm::DenseMap<mlir::Value , double> &timeMap,
```

Comment:
These `mlir::`'s shouldn't be necessary - if you need to, please include circt/Support/LLVM.h to pull them into the circt namespace.

Also, clang format please.